include/deal.II/multigrid/mg_transfer.h

```diff
@@ -154,7 +154,9 @@ class MGLevelGlobalTransfer : public MGTransferBase<VectorType>
 
   /**
    * Transfer from a vector on the global grid to vectors defined on each of
-   * the levels separately, i.a. an @p MGVector.
+   * the levels separately, i.e. a @p MGVector, for the active degrees of
```

Comment:
What are you trying to say with ``MGVector``? I don't think this exists as a type? Do you mean "multilevel vector"?